api/VideoPlaybackQuality.json

```diff
@@ -89,10 +93,10 @@
               "notes": "Only works on Windows 8+."
             },
             "opera": {
-              "version_added": "15"
+              "version_added": "67"
             },
             "opera_android": {
-              "version_added": "14"
+              "version_added": "67"
```

Comment:
Looks like this should also be set to “false”.

Reply:
```
             "opera_android": {
-              "version_added": "14"
+              "version_added": "67"
```
> Looks like this should also be set to “false”.

OK, thanks — made that change